stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -255,6 +255,12 @@ adjoint(B::Bidiagonal) = Adjoint(B)
 transpose(B::Bidiagonal) = Transpose(B)
 adjoint(B::Bidiagonal{<:Real}) = Bidiagonal(B.dv, B.ev, B.uplo == 'U' ? :L : :U)
 transpose(B::Bidiagonal{<:Number}) = Bidiagonal(B.dv, B.ev, B.uplo == 'U' ? :L : :U)
+permutedims(B::Bidiagonal) = Bidiagonal(B.dv, B.ev, B.uplo == 'U' ? 'L' : 'U')
+function permutedims(B::Bidiagonal, perm)
+    Base.checkdims_perm(B, B, perm)
+    all(perm .== (1, 2)) && return B
+    all(perm .== (2, 1)) && return permutedims(B)
+end
```

Comment:
Seems a shame to need broadcasting here, to allow for non-tuple permutations. Would this be safe?
```
julia> @btime all($([1,2]) .== (1,2))
  min 24.866 ns, mean 28.877 ns (2 allocations, 96 bytes)
true

julia> @btime NTuple{2}($([1,2])) == (1,2)
  min 1.292 ns, mean 1.405 ns (0 allocations)
true
```